### DISCUSSION TITLE
Multiple signup per link

### DISCUSSION THREAD

**Discussion by KongGal** [mealie-recipes/mealie#1138] [Feature Request]
**Before submitting this feature request I have**
_COMPLETE AND FILL THESE OUT_
- [x] Reviewed the documentation and verified that my feature request isn't already a feature.
- [x] Reviewed the existing feature requests and my feature has not been requested.
- [x] Checked the [tasks taged](https://github.com/hay-kot/mealie/issues?q=is%3Aissue+is%3Aopen+label%3Atask+) issues and verified my feature is not covered

**Please Describe The Problem To Be Solved**
I would like to see a feature that allows for either unlimited sign up for an sign up invite or a definable number of sign up  
could even upgrade it a bit more and look at how many discord does it and allow for either unlimited sign up or a number of sign up per link but also be able to set a time limit on how long the invite be good for.
This could be connected with a more user based permission system where you can define if a user is member of x group they can then create unlimited, no expiry links or not links at all

**(Optional): Suggest A Solution**
Add a tickbox and a input box  
Tickbox for unlimited (Database limit 0)
Input box for number of sign up possible on the link (defaults to 1 if no number is input or tickbox marked

If there are multiple solutions, please present each one separately. Save comparisons for the very end.)

**Comment by PresWiFi** [comment]
I would also find this very useful!

**Comment by RobCod** [comment]
+1
